src/Lecture2.hs

```diff
@@ -119,7 +143,7 @@ Below is the description of the fight and character specifications:
 
   * A chest contains a non-zero amount of gold and a possible treasure.
     When defining the type of a treasure chest, you don't know what
-    treasures it stores insight, so your chest data type must be able
+    treasures it stores inside, so your chest data type must be able
```

Comment:
Nice catch on the typo! 👀  Feel free to contribute it separately directly to the course repository itself!